src/loose/statement.js

```diff
@@ -446,3 +448,19 @@ lp.parseExportSpecifierList = function() {
   this.popCx()
   return elts
 }
+
+// Set `ExpressionStatement#directive` property for directive prologues.
+lp.adaptDirectivePrologue = function(statements) {
```

Comment:
It seems like you could just call these in the normal parser here, rather than duplicating them.

Reply:
Oh, I have forgotten that `this.toks` is the normal parser.